.github/workflows/phpunit.yml

```diff
@@ -8,31 +8,20 @@ env:
 
 jobs:
   phpunit:
-    name: ${{ matrix.name }}
+    name: "PHP: ${{ matrix.php }} WP: ${{ matrix.wp }}"
```

Comment:
Github is inconsistent about how it uses names in its various UIs. Sometimes it'll combine the workflow and job names, so like "PHPUnit tests - PHP: 7.4 WP: master". In other places it only uses the job name, so like "PHP: 7.4 WP: master".

I'm thinking that the only way to make it sane is to make the job name make sense on its own.